content/en/docs/examples/httpbin-sleep/index.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
This doesn't seem appropriate for the examples section of the doc. It's not an example, but rather setup instructions or some of the other tasks.

Reply:
This is @rcaballeromx's request. I think it's fine. We have something similar for the bookinfo

Reply:
We are planning to clean up the examples section and organize it around proper examples, which this is not. Please remove it and add the commands to start the sleep and httpbin samples in step 4 of your task.

Reply:
When was this decision made and has it been approved by the WG? Please sync with Rigs. I talked to him last week and I had to remove my instructions to create a new page for deploying httpbin and sleep per his request. I'm happy to do it as long as there is an agreed decision.

Reply:
When was what decision made? All I'm saying is that we will be cleaning up the examples section in the near future. We will discuss the details when do it. I'm not sure what Rigs had intended, but there was certainly no WG agreement to turn sleep and httpbin setup instructions into examples. Please make the change I suggested and I will explain it to Rigs if you'd like.